### PR TITLE
Finalize #77423

### LINKED ISSUE #77423: Add title property to QuickPickOptions/InputBoxOptions (showQuickPick)
Currently, `title` can only be set when going through the more complex `createQuickPick` API. It would be great if the title could be set via `QuickPickOptions` when using the simpler `showQuickPick`.

### PR DESCRIPTION
**Pull request by TylerLeonhardt**
This PR fixes #77423

### CODE REVIEW

File: src/vs/vscode.d.ts [hunk 1]
```diff
@@ -1665,6 +1665,12 @@ declare module 'vscode' {
 	 * Options to configure the behavior of the quick pick UI.
 	 */
 	export interface QuickPickOptions {
+
+		/**
+		 * An optional string that represents the tile of the quick pick.
```
Comment (jogo-) [inline src/vs/vscode.d.ts:1670]: tile -> title

File: src/vs/vscode.d.ts [hunk 2]
```diff
@@ -1847,6 +1853,11 @@ declare module 'vscode' {
 	 */
 	export interface InputBoxOptions {
 
+		/**
+		 * An optional string that represents the tile of the input box.
```
Comment (jogo-) [inline src/vs/vscode.d.ts:1857]: tile -> title

**Review by jogo- — COMMENTED** [review]
2 typos in comments

File: src/vs/vscode.d.ts (hunk 1 above)
Comment (TylerLeonhardt) [reply src/vs/vscode.d.ts:1670]: good find!

**Pull request merged**